### PR TITLE
Rate from Book Search

### PR DESCRIPTION
**Pull request by sk1122**
## Description

- Copied rateStart(), rateBook(), finishRate() to startRatingModal(), rateBookModal(), finishRatesModal() with minor changes
    bookid can't be used in layout.html, So using a global variable book__id which will update everytime a Modal opens to the id 
   of Book in Modal

## Affected Dependencies
None

## How has this been tested?
GIF Before

GIF After

## Checklist
- [x] I have followed the Contribution Guidelines.
- [x] My changes do not edit/add any unrequired files.
- [x] My changes are covered by tests.

### CODE REVIEW

File: templates/mainapp/layout.html [hunk 1]
```diff
@@ -336,6 +388,8 @@ <h4 class="modal-title" id="booktitle">Basic Modal</h4>
         // Retrieve and display book details in Modal
         function bookDetails(obj) {
             var bookid = obj.getAttribute('data-book-id');
+            $("#rating-block-modal").attr("data-book-id", bookid);
+            $("#rating-block-modal").addClass("rating-blocsk-" + bookid)
```
Comment (Praful932) [inline templates/mainapp/layout.html:392]: @sk1122 What's the need of this?

**Review by maneprajakta — COMMENTED** [review]
- Check this
bug

**Comment by Praful932** [comment]
@sk1122 Could you update your fork?
We'll fix the code & merge

**Comment by sk1122** [comment]
@Praful932 sorry, i am unable to do it because of college work. I would fix it and push commits again✌️

**Comment by Praful932** [comment]
Hi @sk1122 Is this still being worked upon

**Comment by sk1122** [comment]
Just give me some more days, i am buying a new laptop. I will work on it as soon as possible

**Pull request closed without merging**